libs/application/ui-fields/src/lib/TableRepeaterFormField/TableRepeaterFormField.tsx

```diff
@@ -57,13 +58,34 @@ export const TableRepeaterFormField: FC<Props> = ({
     editButtonTooltipText = coreMessages.editFieldText,
     editField = false,
     maxRows,
+    onSubmitLoad,
   } = data
 
+  const apolloClient = useApolloClient()
+
   const items = Object.keys(rawItems).map((key) => ({
     id: key,
     ...rawItems[key],
   }))
 
+  const load = async () => {
+    if (onSubmitLoad) {
+      try {
+        const submitResponse = await onSubmitLoad({
+          apolloClient,
+          application,
+          tableItems: values,
+        })
+
+        submitResponse.dictinaryOfItems.map((x) => {
```

Comment:
```suggestion
        submitResponse.dictinaryOfItems.forEach((x) => {
```
Since there is no return